tiatoolbox/models/engine/semantic_segmentor.py

```diff
@@ -619,12 +623,13 @@ def get_coordinates(
             >>> segmentor.get_coordinates = func
 
         """
-        (patch_inputs, patch_outputs) = PatchExtractor.get_coordinates(
+        results = PatchExtractor.get_coordinates(
+            patch_output_shape=ioconfig.patch_output_shape,
             image_shape=image_shape,
             patch_input_shape=ioconfig.patch_input_shape,
-            patch_output_shape=ioconfig.patch_output_shape,
             stride_shape=ioconfig.stride_shape,
         )
+        patch_inputs, patch_outputs = results[0], results[1]
```

Comment:
This is redundant. On the next line you can do 

`return results[0], results[1]`